terraform/launch_tmpl.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_launch_template" "whoogle_instance" {
+  name                   = "whoogle-template"
+  image_id               = "ami-094125af156557ca2"
+  instance_type          = "t3.micro"
+  user_data              = filebase64("user_data.sh")
+  vpc_security_group_ids = [aws_security_group.allow_http.id]
+}
+
+resource "aws_autoscaling_group" "whoogleasg" {
+  vpc_zone_identifier = [aws_subnet.public_subnet_1.id, aws_subnet.public_subnet_2.id]
```

Comment:
i think it should be connected to the private subnets

Reply:
👍